libs/sdk-core/src/breez_services.rs

```diff
@@ -2612,6 +2620,18 @@ async fn get_lsp_by_id(
         .cloned())
 }
 
+/// Convenience method to return all LSPs used by this node, active and historical.
+async fn get_used_lsps(
+    persister: Arc<SqliteStorage>,
+    lsp_api: Arc<dyn LspAPI>,
+) -> SdkResult<Vec<LspInformation>> {
+    let node_pubkey = persister
+        .get_node_state()?
+        .ok_or(SdkError::generic("Node info not found"))?
+        .id;
+    lsp_api.list_used_lsps(node_pubkey).await
```

Comment:
I think perhaps out of the historical lsps only we can filter out those that we don't have channels with since we won't get any notifications from them.

Reply:
Done: 4407106d8835a198c28b7a30d3b5a3cbec8317e7